inmem/bucket_service.go

```diff
@@ -209,6 +210,14 @@ func (s *Service) findBuckets(ctx context.Context, filter platform.BucketFilter,
 // FindBuckets returns a list of buckets that match filter and the total count of matching buckets.
 // Additional options provide pagination & sorting.
 func (s *Service) FindBuckets(ctx context.Context, filter platform.BucketFilter, opt ...platform.FindOptions) ([]*platform.Bucket, int, error) {
+	if filter.Name != nil {
```

Comment:
I think we don't need this file. I think this is dead code like the bolt bucket service.